edx_proctoring/backends/software_secure.py

```diff
@@ -99,6 +110,108 @@ def get_software_download_url(self):
         """
         return self.software_download_url
 
+    def on_review_callback(self, payload):
+        """
+        Called when the reviewing 3rd party service posts back the results
+
+        Documentat on the data format can be found from SoftwareSecure's
+        documentation named "Reviewer Data Transfer"
+        """
+
+        log_msg = (
+            'Received callback from SoftwareSecure with review data: {payload}'.format(
+                payload=payload
+            )
+        )
+        log.info(log_msg)
+
+        # payload from SoftwareSecure is a JSON payload
+        # which has been converted to a dict by our caller
+        data = payload['payload']
```

Comment:
Is it ok to have no validation on the payload coming from the SoftwareSecure? Worst case: an exception, but we're logging the payload at the very beginning to help debug. so it might be ok to have no validation. Some validation, however, would make it easier to figure out the exact problem with the payload received (in case there arises one later).